libs/core/models/animeDetails.ts

```diff
@@ -0,0 +1,34 @@
+import { AnimeBase } from './anime';
+import { Genre } from './genre';
+import { OmitImmerable } from './immerable';
+import { Studio } from './studio.dto';
+
+/** Anime details. */
+export class AnimeDetails extends AnimeBase {
+
+  /** List of anime genres. */
+  public readonly genresData: readonly Genre[];
+
+  /** List of anime producing studios. */
+  public readonly studiosData: readonly Studio[];
+
+  /** It's on the air. */
+  public readonly airing: boolean;
```

Comment:
https://wiki.saritasa.rocks/frontend/languages/ts-js/naming#is
```suggestion
  public readonly isAiring: boolean;
```